Assets/MixedRealityToolkit.Tests/EditModeTests/InputSystem/TestFixture_02_InteractionDefinitionTests.cs

```diff
@@ -238,15 +250,29 @@ private static void Test07_TestVector2_internal(bool invertXAxis, bool invertYAx
             i.Vector2Data = vectorValue;
             Vector2 invertedValue = vectorValue * new Vector2(invertXAxis ? -1f : 1f, invertYAxis ? -1f : 1f);
             Assert.AreEqual(invertedValue, i.Vector2Data, msg);
-            Assert.IsTrue(i.Changed, msg);
+            if (vectorValue != Vector2.zero)
+            {
+                Assert.IsTrue(i.Changed, msg);
+            }
+            else
```

Comment:
Nit: It might be helpful to have a comment that this is specifically verifying that changing a value from 0->0 is the single exception that doesn't trigger another change